silkworm/rpc/ws/connection.hpp

```diff
@@ -61,13 +65,20 @@ class Connection : public Channel {
   private:
     Task<void> do_read();
 
+    Task<void> ping_loop();
+
     //! Perform an asynchronous write operation.
     Task<std::size_t> do_write(const std::string& content);
 
     boost::beast::websocket::stream<boost::beast::tcp_stream> ws_;
 
     //! The handler used to process the incoming request.
     http::RequestHandler request_handler_;
+
+    boost::asio::steady_timer retry_timer_;
+
+    //! The retry interval between successive ping attempts
+    static std::chrono::milliseconds ping_interval_;
```

Comment:
At the moment this is not needed, you can remove it and just use the constant in `connection.cpp`.